.github/workflows/module-examples-tests.yaml

```diff
@@ -0,0 +1,71 @@
+name: test-examples
+
+on:
+  workflow_call:
+    secrets:
+      AUTOMATION_TOKEN:
+        required: true
+      SLACK_WEBHOOK:
+        required: true
+
+env:
+  TF_VER: '0.14.11'
+
+jobs:
+  terraform-plan:
+    if: contains(github.event.repository.name, 'aws') && (github.event_name == 'pull_request' && github.event.action == 'synchronize')
```

Comment:
We should also allow for non aws modules, for example the github, and more to follow, what is the alternative?

Reply:
Gor github modules & helm chart module, they require environment variables to be set and a kubernetes cluster respectively, I was wondering if we should consider a separate workflow/job for these.